core/src/com/unciv/logic/automation/city/ConstructionAutomation.kt

```diff
@@ -377,8 +378,8 @@ class ConstructionAutomation(val cityConstructions: CityConstructions) {
         for (unique in building.getMatchingUniques(UniqueType.CarryOverFood, cityState)) {
             if (city.matchesFilter(unique.params[1]) && unique.params[0].toInt() != 0) {
                 val foodGain = cityStats.currentCityStats.food + buildingStats.food
-                val relativeAmount = unique.params[0].toFloat() / 100f
-                buildingStats[Stat.Food] += foodGain * relativeAmount // Essentialy gives us the food per turn this unique saves us
+                val relativeAmount = 1 / ( 1 - unique.params[0].toFloat() / 100f) - 1 
```

Comment:
Sounds like you want .toPercent()

Reply:
X.toFloat() / 100f does the same as X.toPercent() right? I left it like this as the original coder implemented it like this.

Reply:
Let's say the city grows from 20 to 21 pop, it's not just that we save 40% food, or 0.40*220 = 88 food, but when growing then from 21 to 22 pop, we only require 237-88 = 149 food to grow, Thus, our 149 food becomes as valuable as 237 food, an effecive growth increase of 237/149 = 1.59, not 1.40. If we then add a medical lab, the amount of food saved becomes 0.65*220 = 143, so growing from 21 to 22 pop only costs 237-143 = 94 food, which is equivalent to growing by a factor 237/94 = 2.52, multiplying the growth of an aqueduct city by a factor 2.52/1.59 = 1.59. The formula I implemented here is an approximation that doesn't take into account growing citizen food costs and existing multiplyers, but it misjudges the value of aqueducts +17% vs -19%, and the value of medical labs -26% vs -34%, so the error is a bit closer to 0.

Reply:
Ok, let's use a simpler approximation.